src/main/java/md/miller1995/Dealership/services/CarServiceImp.java

```diff
@@ -20,12 +21,23 @@ public CarServiceImp(CarRepository carRepository) {
         this.carRepository = carRepository;
     }
 
-    public List<Car> findAllCars(){
+    public List<CarEntity> findAllCars(){
         return carRepository.findAll();
     }
 
-    public Car findById(long id){
-        Optional<Car> foundCar = carRepository.findById(id);
+    public CarEntity findById(long id){
+        Optional<CarEntity> foundCar = carRepository.findById(id);
         return foundCar.orElse(null);
     }
+
+    @Override
+    public CarDTO CarEntityToCarDTO(CarEntity carEntity) {
```

Comment:
Do we have a method name with an upper case letter?
Also, can't we maybe make use of ObjectMapper for entity -> dto conversion?

Reply:
- changed name of method with lower case letter;
- use modelMapper for conversion entity -> dto